### PR TITLE
Changes to make val-tagged work

### PR DESCRIPTION
**Pull request by epureanudiana**
I made some adjustments to val.py and put them in a new script, val-tagged.py
I needed some adjustments in metrics.py and created a new metrics-tagged.py

The downside is that the 2 classes are almost the same
The upside is that if yolov5 changes, i think it’s easier to just sync

### CODE REVIEW

File: README.zh-CN.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
       <img width="100%" src="https://raw.githubusercontent.com/ultralytics/assets/main/yolov5/v70/splash.png"></a>
   </p>
 
-[英文](README.md)|[简体中文](README.zh-CN.md)<br>
+[英文](YOLOv5_README.md)|[简体中文](README.zh-CN.md)<br>
```
Comment (sdavrieux) [inline README.zh-CN.md:7]: Good morning!! What is this for?

File: data/validation-tagged.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# COCO 2017 dataset http://cocodataset.org
```
Comment (sdavrieux) [inline data/validation-tagged.yaml:1]: What is this file for? An example?

File: README.zh-CN.md (hunk 1 above)
Comment (epureanudiana) [reply README.zh-CN.md:7]: this is their READMEs, I did not want to remove them, but I would like our readme to appear on the front page of the repo, with a link to the ultralytics readme.

File: utils/dataloaders.py [hunk 3]
```diff
@@ -991,6 +996,7 @@ def verify_image_label(args):
     # Verify one image-label pair
     im_file, lb_file, prefix = args
     nm, nf, ne, nc, msg, segments = 0, 0, 0, 0, '', []  # number (missing, found, empty, corrupt), message, segments
+    is_format_tagged = True
```
Comment (sdavrieux) [inline utils/dataloaders.py:999]: Why the default value is True?

File: utils/dataloaders.py [hunk 4]
```diff
@@ -1028,10 +1040,10 @@ def verify_image_label(args):
                     msg = f'{prefix}WARNING ⚠️ {im_file}: {nl - len(i)} duplicate labels removed'
             else:
                 ne = 1  # label empty
-                lb = np.zeros((0, 5), dtype=np.float32)
+                lb = np.zeros((0, 6), dtype=np.float32) if is_format_tagged else np.zeros((0, 5), dtype=np.float32)
         else:
             nm = 1  # label missing
-            lb = np.zeros((0, 5), dtype=np.float32)
+            lb =  np.zeros((0, 6), dtype=np.float32) if is_format_tagged else np.zeros((0, 5), dtype=np.float32)
```
Comment (sdavrieux) [inline utils/dataloaders.py:1046]: There is an extra space here.

File: data/validation-tagged.yaml (hunk 2 above)
Comment (epureanudiana) [reply data/validation-tagged.yaml:1]: yes, the locally used example, I removed it from git

File: utils/dataloaders.py (hunk 3 above)
Comment (epureanudiana) [reply utils/dataloaders.py:999]: the truth value is adjusted later based on what is in the txt file, but I can make this a bit clearer indeed

**Comment by epureanudiana** [comment]
you left me a comment last week: 
`I didn’t really check it yet but I was thinking. In metrics-tagged and val-tagged. Cannot we import from the other files everything that it’s in common and in the new files add only what it’s different?`

I made this change in metrics-tagged, good idea! In val-tagged it does not make sense because the file is essentially a huge run() method which I am changing here and there.

**Review by sdavrieux — APPROVED** [review]

**Pull request merged**